ui/core/src/users/Edit.js

```diff
@@ -0,0 +1,191 @@
+// file: users/Edit.js
```

Comment:
You can use the same user form to edit a user. Please DRY this. Remember, React encourages the _use_ of reusable components.